JobParseAndNormalize.md

```diff
@@ -41,11 +40,11 @@ This service supports the HTTP GET and POST methods. Because Base64-encoded docu
 
 The following parameters may be supplied in the query string (for HTTP GET) or form body (for HTTP POST):
 
-* **document** (Required) -- A .doc, .docx, .pdf, .rtf, .txt, .odt, .wps, and .pages documents given in a BASE64 encoded string.  Please note that the .pages format is not accepted by Textkernel; you will need to specify another parser.
+* **document** (Required) -- A .doc, .docx, .pdf, .rtf, .txt, .odt, .wps, and .pages document given in a BASE64 encoded string.  Please note that the .pages format is not accepted by Textkernel; you will need to specify another parser.
```

Comment:
still need to remove refs to .pages (we only supported those in rpan for daxtra/HA)